app/src/main/java/com/yterletskyi/happyfriend/features/pin/domain/PinCode.kt

```diff
@@ -31,6 +31,20 @@ class LoggedPinCode(
 
 class PinCode(private val maxLength: Int) : Stack<String>() {
 
+    constructor(pin: String) : this(pin.length) {
+        pin.forEach {
+            push(it.toString())
+        }
+    }
+
+    fun getPinCode(): String {
```

Comment:
method name `getPinCode()` in class `PinCode` seems weird, isn't it?
Can we please rename it to something more meaningful like `getStringValue()` or just override `toString()` ?